test/parallel/test-fs-options-immutable.js

```diff
@@ -58,9 +58,7 @@ if (common.canCreateSymLink()) {
   );
 }
 
-if (!common.isAix) {
-  // TODO(thefourtheye) Remove this guard once
-  // https://github.com/nodejs/node/issues/5085 is fixed
+{
```

Comment:
Wouldn't removing the outer block still be consistent with the rest of the test because everything inside that block scope is itself block-scoped? So you'd end up with this block-scoped code?:

```js
{
  let watch;
  assert.doesNotThrow(() => {
    watch = fs.watch(__filename, options, common.noop);
  });
  watch.close();
}

{
  assert.doesNotThrow(() => fs.watchFile(__filename, options, common.noop));
  fs.unwatchFile(__filename);
}
```